### DISCUSSION TITLE
not render PieChart

### DISCUSSION THREAD

**Discussion by alexeevayaan** [Abhinandan-Kushwaha/react-native-gifted-charts#817] [Q&A] [answered]
Hi mans, when my all values in 0, PieChart not rendering^ help plis

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
It is not logical to draw a Pie chart with all values being 0. See https://stackoverflow.com/questions/19534204/is-it-possible-to-show-pie-chart-if-all-data-are-0
